### DISCUSSION TITLE
Elementor weird problem happens time to time/!

### DISCUSSION THREAD

**Discussion by AliBorsan** [elementor/elementor#15092] [Developers Community]
Hi!
Today i couldn't ad elements using Elementor pro, even after clearing cashes and disabling all plugins and trying all kinds of browsers.
when i check my site without elementor opened with chrome dev tools i get no errors in Console.. but when i edit a page with elementor i get lot of error messages and this is one of them:
**`Payment gateway with handle 'wc-payment-method-cod' has been deactivated in Cart and Checkout blocks because its dependency 'wc-blocks-registry' is not registered. Read the docs about registering assets for payment methods: https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/docs/extensibility/payment-method-integration.md#registering-assets`**

**Comment by AliBorsan** [comment]
Okay, i found the problem which is:
Adding passive flag to scroll event handler in JQuery Java script makes you can not add any element using Elementor, and only that can work back after removing the passive block,

**Comment by askwpgirl** [comment]
We are having this same PHP message but even with nothing active but Elementor free version and WooCommerce. 

It has been happening since update to WooCommerce 7.x. So there is still an issue with Elementor. Every time an Elementor page is edited, the logs fill with these PHP messages.
